cmd/gateway-main.go

```diff
@@ -267,7 +267,7 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 	globalLifecycleSys = NewLifecycleSys()
 
 	// Create new notification system.
-	globalNotificationSys = NewNotificationSys(globalServerConfig, globalEndpoints)
+	globalNotificationSys, _ = NewNotificationSys(globalServerConfig, globalEndpoints)
```

Comment:
is it OK to ignore errors here?

Reply:
Yes it will never error out since notification targets are not allowed in gateway @kannappanr